pkg/cmd/pr/create/create.go

```diff
@@ -412,19 +422,22 @@ func initDefaultTitleBody(ctx CreateContext, state *shared.IssueMetadataState, u
 	if err != nil {
 		return err
 	}
-	if len(commits) == 1 || useFirstCommit {
-		commitIndex := len(commits) - 1
-		state.Title = commits[commitIndex].Title
-		body, err := gitClient.CommitBody(context.Background(), commits[commitIndex].Sha)
-		if err != nil {
-			return err
-		}
-		state.Body = body
+
+	if useFirstCommit {
+		state.Title = commits[len(commits)-1].Title
+		state.Body = commits[len(commits)-1].Body
 	} else {
 		state.Title = humanize(headRef)
 		var body strings.Builder
 		for i := len(commits) - 1; i >= 0; i-- {
 			fmt.Fprintf(&body, "- %s\n", commits[i].Title)
+			if addBody {
+				fmt.Fprintf(&body, "    %s", commits[i].Body)
```

Comment:
I see 2 problems with this formatting change:

1. Would want to change this from 3 spaces to 2 spaces in order to align with the title for proper markdown formatting
2. Because many people write larger git commit bodies, there are multiple lines involved and all of it would need to be indented to align.

```
				fmt.Fprintf(&body, "    %s", commits[i].Body)
```

Reply:
I fixed this case also splitting Body every 72 chars, as the best practices for writing commits want... Bonus, the split doesn't split in the middle of the word but it split before the 72 chunk in case is in the middle of word :)